app/core/Router.coffee

```diff
@@ -115,6 +115,8 @@ module.exports = class CocoRouter extends Backbone.Router
     'editor/thang-tasks': go('editor/ThangTasksView')
     'editor/verifier': go('editor/verifier/VerifierView')
     'editor/verifier/:levelID': go('editor/verifier/VerifierView')
+    'editor/i18n-verifier/:levelID': go('editor/verifier/i18nVerifierView')
+    'editor/i18n-verifier': go('editor/verifier/i18nVerifierView')
```

Comment:
Should go over how this works, make sure it's clear. Is this something that'll be useful going forward, if so, what's the expected use case?